Dacaspex/Exhibit
# Fix graph crashing in some cases

#### What have I done
Fixes #384.
The graphservice will not construct whenever an error occurred in the sheet state engine. 
To fix this, I make sure the sheet state engine does not use a position outside the route, which was the cause of the error.

#### Testing
* Open Exhibit
* Do not open the graph component
* Import files
* Observe the graph not crashing.

#### Addresses SRD section(s)
No changes required.
